Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.py

```diff
@@ -0,0 +1,1015 @@
+import traceback
+import dateparser
+from datetime import datetime
+from typing import Dict, Any, Callable, Tuple
```

Comment:
we have these already in Commonserverpython

Reply:
from typing import Callable, Tuple
These aren't in Commonserverpython